engine/setup/CMakeLists.txt

```diff
@@ -22,10 +22,22 @@
 # along with Vega Strike. If not, see <https://www.gnu.org/licenses/>.
 #
 
-CMAKE_MINIMUM_REQUIRED(VERSION 3.16 FATAL_ERROR)
+CMAKE_MINIMUM_REQUIRED(VERSION 3.21 FATAL_ERROR)
+
+IF (POLICY CMP0087)
+	cmake_policy(SET CMP0087 NEW)
+ENDIF ()
+SET(X_VCPKG_APPLOCAL_DEPS_INSTALL ON)
+
+IF (POLICY CMP0167)
+	CMAKE_POLICY (SET CMP0167 OLD)
+ENDIF (POLICY CMP0167)
```

Comment:
This is set i the root Cmakelists.txt and shouldn't be needed here.

Reply:
Fixed